### ISSUE TITLE
Parking post height adjustment

### ISSUE THREAD

**Issue by sgbaird** [machineagency/science-jubilee#153]
Mention the height adjustment required when mounting a new tool in the tools docs, i.e., tool positioning

**Comment by brendenpelkie** [comment]
Yes this is important to add. Our documentation would benefit from a re-written tool parking post position tutorial that includes z-axis adjustment and other tweaks. I'll work on adding this.

**Comment by sgbaird** [comment]
Forgot to add: https://youtube.com/shorts/_3stUKidY7Y?si=29VHJ1fLYcNBsfVH and https://youtube.com/shorts/EEWD-wCFl9g?si=dWMmSwy5AnTrTyrD can be added to help address this
